blocks/library/gallery/index.js

```diff
@@ -7,7 +7,7 @@ import { filter, every } from 'lodash';
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
-import { createMediaFromFile, preloadImage } from '@wordpress/utils';
+import { mediaUpload } from '@wordpress/utils';
```

Comment:
It would be nice to double check if we still need to export `createMediaFromFile` from utils.

Reply:
Yes, there is no need to export it, and the function should not be used directly so the export was removed.